tools/buildtools/ambertools/parmconv.xml

```diff
@@ -11,14 +11,53 @@
   </expand>
   <command detect_errors="exit_code">
     <![CDATA[
-   python '$templating_script' '$inputs' &&
-   PATH_TO_PARMED=\$(dirname `which parmed`) &&
-   export AMBERHOME=\$(dirname \$PATH_TO_PARMED) &&
-   export GMXDATA=\$AMBERHOME/share/gromacs/top/ &&
-   parmed -i ligand.script -O &&
-   parmed -i receptor.script -O &&
-   parmed -i complex.script -O &&
-   parmed -i solvatedcomplex.script -O
+   #if '$param_inputs.modbehaviour.removedihe' or '$param_inputs.modbehaviour.removebox':
+     python '$__tool_directory__/parmed_new_convert.py' 
+       --istr '$param_inputs.str_in'
+       --itop '$param_inputs.top_in'
+       $param_inputs.modbehaviour.removedihe
+       $param_inputs.modbehaviour.removebox
+       --iradii '$param_inputs.modbehaviour.radii' 
+       --istripmask '$stripmask_ligand'
+       --o_prmtop '$prmtop_ligand'
+     2>&1  && 
+     python '$__tool_directory__/parmed_new_convert.py' 
+       --istr '$param_inputs.str_in'
+       --itop '$param_inputs.top_in'
+       $param_inputs.modbehaviour.removedihe
+       $param_inputs.modbehaviour.removebox
+       --iradii '$param_inputs.modbehaviour.radii' 
+       --istripmask '$stripmask_receptor'
+       --o_prmtop '$prmtop_receptor'
+     2>&1 &&
+     python '$__tool_directory__/parmed_new_convert.py' 
+       --istr '$param_inputs.str_in'
+       --itop '$param_inputs.top_in'
+       $param_inputs.modbehaviour.removedihe
+       $param_inputs.modbehaviour.removebox
+       --iradii '$param_inputs.modbehaviour.radii' 
+       --istripmask '$stripmask_complex'
+       --o_prmtop '$prmtop_complex'
+     2>&1 &&
+     python '$__tool_directory__/parmed_new_convert.py' 
+       --istr '$param_inputs.str_in'
+       --itop '$param_inputs.top_in'
+       $param_inputs.modbehaviour.removedihe
+       $param_inputs.modbehaviour.removebox
+       --iradii '$param_inputs.modbehaviour.radii' 
+       --istripmask '$stripmask_solvatedcomplex'
+       --o_prmtop '$prmtop_solvatedcomplex'
+     2>&1
+   #else
```

Comment:
@chrisbarnettster do you think this is still needed, or can the `parmed_new_convert.py` script deal with this as well?

Reply:
Yes the new script can deal with this. have you made changes or should I go ahead?

Reply:
Is this okay for you: https://github.com/galaxycomputationalchemistry/galaxy-tools-compchem/pull/75/commits/ec30a7dda93f87f906dbba9f0fcadfe240e912b5

I'll write a test now as well

Reply:
Yes looks good. I had kept the parmed calls as a 'fail safe' while testing. Happy to remove.